### PR TITLE
unified spaces arround brackets

### PR DESCRIPTION
**Pull request by NilsKasper**
unified spaces arround brackets

### CODE REVIEW

File: src/MensaWidget/MWMenuBar.class.st [hunk 1]
```diff
@@ -168,11 +168,11 @@ MWMenuBar >> decreaseDate: aEvent [
 
 {
 	#category : #dateManipulation,
-	#'squeak_changestamp' : 'Louis Christ 6/9/2024 15:50'
+	#'squeak_changestamp' : 'NK 6/18/2024 21:33'
 }
 MWMenuBar >> increaseDate [
 
-	self date: (self date addDays: 1);
+	self date: (self date addDays: 1) ;
```
Comment (lchristus) [inline src/MensaWidget/MWMenuBar.class.st:175]: we dont need the space here

File: src/MensaWidget/MWFrame.class.st [hunk 2]
```diff
@@ -126,7 +126,7 @@ MWFrame >> buildScrollPane [
 			extent: self class contentExtent;
 			hResizing: #spaceFill;
 			vResizing: #spaceFill;
-			yourself);
+			yourself) ;
```
Comment (lchristus) [inline src/MensaWidget/MWFrame.class.st:129]: no space here

File: src/MensaWidget/MWMenuBar.class.st [hunk 3]
```diff
@@ -30,12 +30,12 @@ Class {
...
 {
 	#category : #constants,
-	#'squeak_changestamp' : 'Louis Christ 6/10/2024 17:41'
+	#'squeak_changestamp' : 'NK 6/18/2024 21:34'
 }
 MWMenuBar class >> getArrowButton [
 
 	^ PolygonMorph new
-		setVertices: {0 @ 0. 0 @ 20. 20 @ 10};
+		setVertices: {0 @ 0. 0 @ 20. 20 @ 10} ;
```
Comment (lchristus) [inline src/MensaWidget/MWMenuBar.class.st:38]: no space here

File: src/MensaWidget/MWMenuBar.class.st [hunk 4]
```diff
@@ -146,11 +146,11 @@ MWMenuBar >> dateDisplay: anObject [
 
 {
 	#category : #dateManipulation,
-	#'squeak_changestamp' : 'Louis Christ 6/9/2024 15:50'
+	#'squeak_changestamp' : 'NK 6/18/2024 21:33'
 }
 MWMenuBar >> decreaseDate [
 
-	self date: (self date addDays: -1);
+	self date: (self date addDays: -1) ;
```
Comment (lchristus) [inline src/MensaWidget/MWMenuBar.class.st:153]: no space here

**Review by lchristus — CHANGES_REQUESTED** [review]
Very nice I guess I should have clarfied further we dont need a space when having a following semicolon but otherwise perfect

**Review by Kangaroo6942 — APPROVED** [review]
Looks good, I found no unfixed methods that aren't going to be fixed on another branch

**Review by lchristus — APPROVED** [review]

**Review by Kangaroo6942 — APPROVED** [review]

**Review by tommccaughey — APPROVED** [review]

**Review by lchristus — APPROVED** [review]

**Pull request merged**